spec/controllers/api/v1/monsters_controller_spec.rb

```diff
@@ -5,7 +5,7 @@
   let!(:second_monster) { FactoryBot.create(:monster) }
 
   describe "GET#index" do
-    it "should return a list of monsters and their images and rating" do
+    it "should return a list of monsters and categorization" do
```

Comment:
Looks like we're testing the monsters and their description, I don't see anything being tested about categorization?